package.json

```diff
@@ -43,5 +43,8 @@
   },
   "devDependencies": {
     "@vaadin/vaadin-component-dev-dependencies": "^3.0.0"
+  },
```

Comment:
Vaadin icons dependency should not be added here.

Reply:
Done